composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "ezsystems/ezpublish-kernel": "^6.8@beta",
+        "ezsystems/ezpublish-kernel": "^6.8",
```

Comment:
does it work with 6.7 as well? _(no stable 6.8 yet so this won't resolve unless forced)_